Samples/UserActivity/cs/SampleConfiguration.cs

```diff
@@ -66,7 +69,8 @@ protected override void OnActivated(IActivatedEventArgs e)
                 var protocolArgs = (IProtocolActivatedEventArgs)e;
                 foreach (var scenario in mainPage.Scenarios)
                 {
-                    if (scenario.ClassType.Name == protocolArgs.Uri.Query)
+                    var query = (protocolArgs.Uri.Query[0] == '?') ? protocolArgs.Uri.Query.Substring(1) : protocolArgs.Uri.Query;
```

Comment:
Still doesn't work if there is no query at all. (`[0]` generates an invalid index exception). I'll fix this too.